0.2.0/gremlin-server.yaml

```diff
@@ -0,0 +1,39 @@
+host: 0.0.0.0
+port: 8182
+threadPoolWorker: 1
+gremlinPool: 8
+scriptEvaluationTimeout: 30000
+serializedResponseTimeout: 30000
+channelizer: org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer
+graphs: { graph: /etc/opt/janusgraph/janusgraph.properties }
```

Comment:
Do we want to support the option of using ConfiguredGraphFactory? I think defaulting to JanusGraphFactory is fine for 0.2.x, but we'll probably want CGF for 0.3.x and above.